LICENSE

```diff
@@ -1,339 +1,13 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 2, June 1991
 
- Copyright (C) 1989, 1991 Free Software Foundation, Inc.,
- 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
- Everyone is permitted to copy and distribute verbatim copies
- of this license document, but changing it is not allowed.
+wolfCLU is either licensed for use under the GPLv2 (or at your option any
```

Comment:
Is the "GNU GENERAL PUBLIC LICENSE" text included somewhere as well? Maybe in a gplv2.txt and reference it here?